SimplyPluralApiCommons.yml

```diff
@@ -17,21 +17,25 @@ components:
       type: object
       properties:
         color:
-          description: Color of the object (\#ffffff, \#ffffffff or ffffff)
+          description: 'Color of the object in a hex format (eg: \#ffffff, \#ffffffff or ffffff)'
         private:
           description: Whether this object is private
         preventTrusted:
           description: Wheter trusted friends are prevented from seeing this private object
         supportDescMarkdown:
           description: Whether the description supports markdown
         timestamp:
-          description: Milliseconds since epoch for this timestamp/date
+          description: Time in Milliseconds since epoch
+        uid:
+          description: The system Id
+        lastOperationTime:
+          description: Time in milliseconds since epoch of the last update on this object
 
   parameters:
     objectId:
```

Comment:
create a separate parameter (objectIdOptional), or something akin to that. Then set required to false. The POST requirements of an ID must match a mongo objectid, backwards compatibility does not apply to POST requests. Then use this separate parameter for all POST endpoints that have optional IDs (Which is most of them)